plinth/modules/jsxc/__init__.py

```diff
@@ -0,0 +1,97 @@
+#
+# This file is part of Plinth.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+"""
+Plinth module to configure XMPP web client/jsxc
+"""
+
+from django.urls import reverse_lazy
+from django.utils.translation import ugettext_lazy as _
+import logging
+import socket
+
+from plinth import actions
+from plinth import action_utils
+from plinth import cfg
+from plinth import frontpage
+from plinth import service as service_module
+from plinth.signals import pre_hostname_change, post_hostname_change
+from plinth.signals import domainname_change
+
+
+version = 2
+
+depends = ['apps']
+
+managed_packages = ['libjs-jsxc']
+
+title = _('JSXC')
+
+description = [
+
+    _('To actually communicate, you can use the web client or any other '
```

Comment:
How about this:
"JSXC is a web client for XMPP. Typically it is used with an XMPP server running locally."